src/bmp.imageio/bmpinput.cpp

```diff
@@ -99,6 +99,10 @@ BmpInput::open(const std::string& name, ImageSpec& spec)
         m_padded_scanline_size = (m_spec.width + 3) & ~3;
         if (!read_color_table())
             return false;
+        // m_allgray is a new BMPInput data member, initialized to false
```

Comment:
This comment can be eliminated, it was just explaining what m_allgray was for when I was explaining this idea in the ticket

Reply:
Removed in latest commit.